### PR TITLE
Update to 2.4

### PR DESCRIPTION
**Pull request by mariusvniekerk**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
@conda-forge-admin , please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {% set name = "pigz" %}
-{% set version = "2.3.4" %}
+{% set version = "2.4" %}
 {% set sha256 = "6f031fa40bc15b1d80d502ff91f83ba14f4b079e886bfb83221374f7bf5c8f9a" %}
```
Comment (jakirkham) [inline recipe/meta.yaml]: New checksum?

Comment (mariusvniekerk) [reply recipe/meta.yaml]: yep

**Comment by epruesse** [comment]
Are you still working on this?

**Comment by mariusvniekerk** [comment]
Feel free to take it over

**Pull request closed without merging**